### DISCUSSION TITLE
Enable or Disable Tag immutability

### DISCUSSION THREAD

**Discussion by phrutos** [TechToSpeech/terraform-aws-serverless-static-wordpress#77] [General]
Hi there, thanks for sharing this method. I am trying to set up everything from scratch and wonder if this should be on or off. Thanks!

Tag immutability
Enable tag immutability to prevent image tags from being overwritten by subsequent image pushes using the same tag. Disable tag immutability to allow image tags to be overwritten.
Enable/Disabled?

**Comment by petewilcock** [comment]
Hi @phrutos, it is best to have tag immutibility off. Subsequent runs of codebuild push and overwrite `latest` tag by default.

**Reply by phrutos** [reply]
Awesome! Thanks. I'm far from getting all the setup correct, so I may bother you with a few other things. Should I break it down through multiple discussion topics?

**Reply by petewilcock** [reply]
You can ask a few general questions on a thread, no problem.

**Reply by phrutos** [reply]
Ok, great. Thanks! This will be experimentation, which I will document on my end and I will probably create a YouTube tutorial and reference your repo to help others do the same thing. So, if you bare with me while I learn my way around all the tools and setups, this could benefit more people.
